src/rpdk/core/data_loaders.py

```diff
@@ -148,6 +141,28 @@ def load_resource_spec(resource_spec_file):  # noqa: C901
         LOG.debug("Resource spec validation failed", exc_info=True)
         raise SpecValidationError(str(e)) from e
 
+    if {
+        "maxresults",
+        "maxrecords",
+        "maxitems",
+        "nexttoken",
+        "nextmarker",
+        "nextpagetoken",
+        "pagetoken",
+    } & set(map(str.lower, resource_spec.get("properties", []))):
+        LOG.warning(
+            "LIST API inputs like MaxResults, MaxRecords, MaxItems, NextToken, NextMarker, NextPageToken, PageToken, and Filters are not resource properties"
```

Comment:
comment got lost since I force pushed, but [not confident enough `Filters` alone should trigger this warning, but wanted to include `Filters` in the warning message that's triggered if they have any of the others](https://github.com/aws-cloudformation/cloudformation-cli/commit/3828495763792996d38e76ff4e0a174e03f66ed8#r45939504) since a couple teams incorrectly had `Filters` already:

https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-licensemanager/pull/3
https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-ssm/pull/75

Reply:
but if a customer sets filter and none of the above list it won't trigger right? could still log the ones that do before appending filters to the end right?

```
" {} and Filters ...".format(set_intersection_results)
```
or whatever the formatting stuff is for logs